HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py

```diff
@@ -20,7 +20,7 @@
         'keep triggerTriggerEvent_*_*_*',
         'keep *_hltFEDSelectorL1_*_*',
         'keep *_hltScoutingEgammaPacker_*_*',
-        'keep *_hltScoutingMuonPacker_*_*',
+        'keep *_hltScoutingMuonPacker*_*_*',
```

Comment:
@drkovalskyi this is not the way we are supposed to change the event content. Please open a JIRA ticket and STORM will take care of changing it centrally via updating confDB first.